make/RunTests.gmk

```diff
@@ -1361,18 +1362,23 @@ ifeq ($(TEST_OPTS_JCOV), true)
 	$(JAVA) -jar $(JCOV_HOME)/lib/jcov.jar GrabberManager -stop -stoptimeout 3600
 
   JCOV_REPORT_TITLE := JDK code coverage report<br/>
+  ifneq ($(JCOV_MODULES), )
+    JCOV_MODULES_FILTER := $(foreach m, $(JCOV_MODULES), -include_module $m)
+    JCOV_REPORT_TITLE += Included modules: $(JCOV_MODULES)<br>
+  endif
   ifneq ($(JCOV_FILTERS), )
     JCOV_REPORT_TITLE += Code filters: $(JCOV_FILTERS)<br>
   endif
   JCOV_REPORT_TITLE += Tests: $(TEST)
 
   jcov-gen-report: jcov-stop-grabber
 	$(call LogWarn, Generating JCov report ...)
-	$(JAVA) -Xmx4g -jar $(JCOV_HOME)/lib/jcov.jar RepGen -sourcepath \
+	$(call ExecuteWithLog, $(JCOV_SUPPORT_DIR)/run-jcov-repgen,
```

Comment:
Shouldn't there be a backslash at the end of this line?

Reply:
```
Generating JCov report ...
RunTests.gmk:1378: *** unterminated call to function 'call': missing ')'.  Stop.
make[2]: *** [make/Main.gmk:820: jcov-test] Error 2

```

Reply:
It should indeed